descholar-ceo/find-flight-info
# User sees navbar

#### What does this PR ​ do​ ?
It allows users to see tha navigation bar which helps them to navigate through the app so easily


#### Description of Task to be completed?
* -Create component of navbar
*  -Add redux for state management
*  -Change login system to use redux store

#### How should this be manually tested?
* Clone this repository by `git clone https://github.com/descholar-ceo/find-flight-info`
* Type these commands : 
    *  `cd find-flight-info` to go in the root directory of the project
    * `git pull origin ft-navbar` to pull this branch
    * `npm install` or `yarn add` to install all the packages
    * `npm start` or `yarn start`
* Then go in the browser, you will see a default opened page is signup with the navigation bar above

#### Any background context you want to provide?
* You have to install [nodejs](https://nodejs.org/en/) version 6 or higher prior to test this application

#### What are the relevant pivotal tracker stories?

#### Screenshots (​ if​ appropriate)
![05 navbar](https://user-images.githubusercontent.com/56886064/75113619-2a98f700-5658-11ea-8780-d5e1e50b61bc.png)

![06 navbar](https://user-images.githubusercontent.com/56886064/75113683-92e7d880-5658-11ea-808f-e17b3fd72c4b.png)


#### Questions:
No Question
